doc/nrf/known_issues.rst

```diff
@@ -1087,14 +1088,14 @@ DRGN-15852: In rare cases on nRF53 Series, the scanner generates corrupted adver
   * TX_Power
   * Advertising_SID
 
-.. rst-class:: v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
+.. rst-class:: v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
```

Comment:
this was also fixed in sdc for 1.6.1
https://github.com/nrfconnect/sdk-nrf/pull/5122
https://github.com/nrfconnect/sdk-nrfxlib/pull/517